src/org/zaproxy/zap/extension/replacer/ReplacerParamRule.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development Team
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.replacer;
+
+import java.util.List;
+
+import org.zaproxy.zap.utils.Enableable;
+
+class ReplacerParamRule extends Enableable {
+
+    public enum MatchType {req_header, req_header_str, req_body_str, resp_header, resp_header_str, resp_body_str}; 
```

Comment:
The enum names should be all upper case (the white space at the end of the line could be removed, as well the semicolon).